### PR TITLE
Add new cloudformation templates as per alert logic docs

### LINKED ISSUE #5: Update alert logic appliances
CentOS has announced the end of life for their version 6 and 8 operating systems. This means CentOS will no longer provide security patches, vulnerabilities, or bug fixes for these operating systems. As a result, Alert Logic is helping customers update all Alert Logic appliances running on out-of-date operating systems.

Ref : https://docs.alertlogic.com/prepare/alma-linux-update-mdr-aws.htm

### PR DESCRIPTION
**Pull request by Chithrak07**

### CODE REVIEW

**Review by dale-c-anderson — APPROVED** [review]

File: cloudformation/al_security_ids_full.template [hunk 1]
```diff
@@ -45,7 +45,7 @@
         "InstanceType": {
             "Description": "Select the instance type to launch IDS security appliances",
             "Type": "String",
-            "Default": "c5.large",
+            "Default": "c5.xlarge",
```
Comment (dale-c-anderson) [inline cloudformation/al_security_ids_full.template:48]: @Chithrak07 I just noticed this - we've doubled the size of all these instances, which is causing overages for all the projects using them. 

Is this a typo? or is the new default actually xlarge?

Comment (Chithrak07) [reply cloudformation/al_security_ids_full.template:48]: yes, the default is not c5.xlarge. https://docs.alertlogic.com/deploy/aws-manual-pro-ent.htm#Deployscanningappliances
Should I revert this?

Comment (dale-c-anderson) [reply cloudformation/al_security_ids_full.template:48]: We need to find out from alert logic if xlarge is actually necessary.

Comment (Chithrak07) [reply cloudformation/al_security_ids_full.template:48]: Support ticket created.

**Pull request merged**